### ISSUE TITLE
Running Leftover bugs for later

### ISSUE THREAD

**Issue by duskfallcrew** [Ktiseos-Nyx/Ktiseos-Nyx-Trainer#193]
These are from the PR suggestions that @coderabbit gave, and i'm doing this so i can push/merge to main without losing stuff

Vendored code to poke:

@duskfallcrew, results are clear now — this comes from upstream LyCORIS v3.2.0.dev11, not from your fork. Both files have the pattern:

kohya.py Line 1031: 'lr': torch.tensor(current_lr)
wrapper.py Line 851: param_data["lr"] = torch.tensor(lr)
So this is a perfect candidate to fix locally in your vendor copy and submit upstream as a PR to [KohakuBlueleaf/LyCORIS](https://github.com/KohakuBlueleaf/LyCORIS). The fix is two trivial one-liners:

trainer/derrian_backend/lycoris/lycoris/kohya.py — Line 1031:

# Before
'lr': torch.tensor(current_lr),

# After
'lr': float(current_lr),
trainer/derrian_backend/lycoris/lycoris/wrapper.py — Line 851:

# Before
param_data["lr"] = torch.tensor(lr)

# After
param_data["lr"] = float(lr)
Using float() rather than just removing torch.tensor() bare is slightly safer since current_lr could theoretically already be a tensor (e.g., from a LoRA+ multiplication), so float() handles both cases cleanly. This would protect against the DAdapt/Prodigy logging issue and any HuggingFace scheduler type mismatches we identified earlier.

**Comment by duskfallcrew** [comment]
All these were scraped from https://github.com/Ktiseos-Nyx/Ktiseos-Nyx-Trainer/pull/190

**Comment by duskfallcrew** [comment]
Still missing wiring up for:
 
  Also worth noting: the frontend training job validator only accepts ['sd15', 'sdxl', 'sd3', 'flux', 'lumina'] but the
  form schema includes Anima, HunyuanImage, Chroma, and SD3.5 too — those will need syncing when you're ready to wire
  them up.


Base URLs for downloads i'll do in a seperate issue

**Comment by duskfallcrew** [comment]
Ok cool, that should be everything ffs this was a huge one someone please PLEASE send me on vacation XD
